BenchmarkSample/benchmark/build.gradle

```diff
@@ -3,19 +3,20 @@ apply plugin: 'kotlin-android'
 apply plugin: 'androidx.benchmark'
 
 android {
-    compileSdkVersion 29
+    compileSdkVersion 31
 
     defaultConfig {
         minSdkVersion 14
-        targetSdkVersion 29
+        targetSdkVersion 31
 
         // Set this argument to capture profiling information, instead of measuring performance.
         // Can be one of:
         //   * None
         //   * Sampled
         //   * Method
         // See full descriptions of available options at: d.android.com/benchmark#profiling
-        testInstrumentationRunnerArgument 'androidx.benchmark.profiling.mode', 'none'
+        // TODO for benchmark 1.1.0-alpha02 and higher, when no benchmarking profiling needed, remove from gradle
```

Comment:
what is this comment here for? I'm not clear what the TODO is

the idea here is that people can change "none" to something else if they want profiling